Project.toml

```diff
@@ -4,16 +4,21 @@ version = "0.9.0-dev"
 
 [deps]
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
-Requires = "ae029012-a4dd-5104-9daa-d747884805df"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
+[weakdeps]
+Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
+
+[extensions]
+FixedPointNumbersStatisticsExt = "Statistics"
+
 [compat]
-Requires = "1"
 julia = "1"
 
 [extras]
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [targets]
-test = ["Documenter", "Test"]
+test = ["Documenter", "Statistics", "Test"]
```

Comment:
I don't fully understand how Package Extension works, but this package still hardly `[dep]`ends on `Statistics` for the time being, so I guess it is not needed for `[extras]` and `test`.